### PR TITLE
fix(button): avoid direct access to nativeElement for consistency

### PR DESCRIPTION
**Pull request by codef0rmer**

### CODE REVIEW

File: src/material/button/button.ts [hunk 1]
```diff
@@ -98,7 +98,7 @@ export class MatButton extends _MatButtonMixinBase
     // attributes, add the correct corresponding class.
     for (const attr of BUTTON_HOST_ATTRIBUTES) {
       if (this._hasHostAttributes(attr)) {
-        (elementRef.nativeElement as HTMLElement).classList.add(attr);
+        this._getHostElement().classList.add(attr);
```
Comment (jelbourn) [inline src/material/button/button.ts]: Could you also add a return type of `HTMLElement` to `_getHostElement` in this PR? That will ensure that the type safety remains

Comment (codef0rmer) [reply src/material/button/button.ts]: Done :-)

**Comment by jelbourn** [comment]
Commit message should be:
```
chore(button): re-use common function for getting native element
```

**Comment by codef0rmer** [comment]
Done.

**Pull request merged**